STYLE_API_TEMPLATE.md

```diff
@@ -0,0 +1,142 @@
+  *Example usage*: 
+  The response provides a value for ```time_field``` , such as “timestamp.” 
+
+See also [Examples](https://alpha-docs-aws.amazon.com/awsstyleguide/latest/styleguide/examples.html) in the AWS Style Guide.
+
+## Basic elements for documentation
+The basic structure for API documentation is informed by the following elements, each discussed under their respective headings below.
+
+When formatting, follow the heading hierarchy presented here. Depending on where the documentation appears within a section or subsection, heading levels may be adjusted to fit with other content.
+
+1. Name of API (heading level 2)
+2. Path parameters (heading level 3)
+3. Query parameters (heading level 3)
+4. Sample request (heading level 4)
+5. Sample response (heading level 4)
+6. Request fields (heading level 3)
+7. Sample request (heading level 4)
+
+## API name
+
+Provide an API name that describes its function, followed by a description of its top use case and any usage recommendations.
+
+*Example function*: "Autocomplete queries"
+
+Use sentence capitalization for the heading (for example, "Create or update mappings"). When you refer to the API operation, you can use lowercase with code font.
+
+If there is a corresponding OpenSearch Dashboards feature, provide a “See also” link that references it. 
+*Example*:  “To learn more about monitor findings, see [Document findings](https://opensearch.org/docs/latest/monitoring-plugins/alerting/monitors/#document-findings)."
+
+If applicable, provide any caveats to its usage with a note or tip, as in the following example:
+
+"If you use the security plugin, make sure you have the appropriate permissions."
+(To set this point in note-style format, follow the text on the next line with {: .note })
+
+### Path parameters
+
+While the API endpoint states a point of entry to a resource, the path parameter identifies a specific resource within it. Path parameters are located in the path of the endpoint just before the query string and following the resource name in the URL.
+
+
+```json
+GET _search/scroll/<scroll_id>
+```
+In the example above, the endpoint is "scroll" and its path parameter is "scroll_id".
+
+Introduce what the path parameters can do at a high level. Provide a table with parameter names and descriptions. Include a table with the following columns:
+*Parameter* - Parameter name in plain font.
+*Data Type* - Data type capitalized (such as Boolean, String, or Integer).
+*Description* - Sentence to describe the parameter function, default values or range of values, and any usage examples.
+
+Parameter | Data Type | Description
+:--- | :--- | :---
+
+
+### Query parameters
+
+In terms of placement, query parameters are always appended to the end of the URL and located to the right of the operator "?". Query parameters serve the purpose of sorting and/or filtering information to be retrieved from the resource. 
```

Comment:
Not necessarily only sorting and filtering. The verbose parameter below just makes the output more verbose, for example. So, I would change "sorting and/or filtering" to  "modifying".